Functions/Mock.ps1

```diff
@@ -1381,7 +1381,7 @@ function Get-DynamicParametersForCmdlet
         return
     }
 
-    if ($PSVersionTable.PSVersion -ge '5.0.10586.122')
+    if ('5.0.10586.122' -le $PSVersionTable.PSVersion)
```

Comment:
You reverted logic of the program flow -- means the code for newer version will be executed for older.

Reply:
If the PSVersionTable.PSVersion was 6.0 (for example)

6.0 -ge 5.0 = true //original logic
5.0 -le 6.0 = true //new logic

How would this change the program flow if the conditional output remains the same, given both variables?

Reply:
Sorry, you are right. It's my fault. I need sleep more :-(